### PR TITLE
update slumber fork version.

### PR DESCRIPTION
**Pull request by jimabramson**
@rlucioni this never gets old, does it.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,14 +2,17 @@
...
     packages=['ecommerce_api_client'],
     url='https://github.com/edx/ecommerce-api-client',
     description='Client used to access edX E-Commerce Service',
     long_description=open('README.rst').read(),
     install_requires=[
-        'slumber==0.7.0',
+        'slumber==0.7.1-decode-utf8',
     ],
+    dependency_links=[
+        'git+https://github.com/edx/slumber.git@0.7.1-decode-utf8#egg=slumber',
```
Comment (rlucioni) [inline setup.py]: Is it necessary to specify a version here, too?
```python
'git+https://github.com/edx/slumber.git@0.7.1-decode-utf8#egg=slumber==0.7.1-decode-utf8',
```

**Comment by jimabramson** [comment]
updated, fixed missing comma.

File: setup.py (hunk 1 above)
Comment (jimabramson) [inline setup.py]: i believe that is redundant because it's specfied in `install_requires`

Comment (rlucioni) [inline setup.py]: Okay, thanks for the clarification.

**Comment by rlucioni** [comment]
:+1:

**Comment by jimabramson** [comment]
@rlucioni @clintonb @feanil heads up.  turns out it's impossible to install a dependency via setup.py from a vcs or any remote link.  so edx-platform is responsible for installing it via requirements.txt.  (see: https://github.com/pypa/pip/pull/1519)

**Comment by feanil** [comment]
What is the corresponding edx-platform PR?

**Comment by jimabramson** [comment]
@feanil https://github.com/edx/edx-platform/pull/8032

**Comment by clintonb** [comment]
:+1:

**Comment by feanil** [comment]
:+1:

**Pull request merged**